test/Microsoft.ML.AutoML.Tests/ApprovalTests/TrainerExtensionsTests.BuildDefaultAveragedPerceptronPipelineNode.approved.txt

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "AveragedPerceptronBinary",
+  "NodeType": 1,
+  "InColumns": [
+    "Features"
+  ],
+  "OutColumns": [
+    "Score"
+  ],
+  "SerializedProperties": {
+    "LabelColumnName": "\"L\"",
```

Comment:
Are the extra quotes expected?

Reply:
What if the column name has its own quotes in it? Does it end up double quoted?

For instance, a user choosing to name their label field as: `Length ("inches")`

Reply:
good point, thanks for point out

Reply:
The new encode rule will be `@"value.replace("\"", "\"\"")"`. So if value is `("inches")`, the corresponding code will be `@"(""inches"")"`, and in json it will be encoded as `"@\"(\"\"inches\"\")\"`.

Reply:
### Resolved